KVeitch/bring-your-own-backend
# Creates and connect initial front-end files

#### What's this PR do?
Creates a mls2019-sandbox folder and creates a basic react add in the directory.
Changes the favicon to be the MLS logo from the react logo.
Connect the initial components.
Creates a utils folder.
Creates apiCalls.js in utils folder.
getTeams, getTeam, getPlayer, getPlayers, getRoster functions created in apiCalls.js

#### Where should the reviewer start?  
mls2019-sandbox/src/utils/apiCalls.js

#### How should this be manually tested?
1. Pull down and switch to the mls2019-sandbox directory.
2.  ```npm install```
3. ```npm start```
4. check for copmponents connection



#### Questions:

- Does the knowledge base need an update?
No
- Does this add new dependencies which need to be added to chef?
react


- [x] Finish my changes
- [x] Push my commits to GitHub
- [x] Open a pull request
- [x] \(Optional) Open a followup issue
